ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -536,6 +587,7 @@ void OwInterface::initialize()
     m_digCircularClient = std::make_unique<DigCircularActionClient>(Op_DigCircular, true);
     m_digLinearClient = std::make_unique<DigLinearActionClient>(Op_DigLinear, true);
     m_deliverClient = std::make_unique<DeliverActionClient>(Op_Deliver, true);
+    m_identifySampleLocationClient = std::make_unique<IdentifySampleLocationActionClient>(Op_IdentifySampleLocation, true);
```

Comment:
Please shorten this line (by splitting it) for better readability -- 120 is max in our coding guidelines.  I like to keep lines as close to 80 as possible, though this is an old tradition :)